dotCMS/src/main/java/com/dotmarketing/quartz/job/ResetPermissionsJob.java

```diff
@@ -90,6 +91,7 @@ public ResetPermissionsJob() {
      * @param jobContext - The {@link JobExecutionContext} containing details and execution
      *        parameters of the job.
      */
+	@CloseDBIfOpened

```

Comment:
Annotate with `@WrapInTransaction` instead and remove all the `HibernateUtil.startTransaction();` , `HibernateUtil.closeAndCommitTransaction();` and also remove the `try` `catch` and `finally`